modules/statics/src/networks.ts

```diff
@@ -484,8 +484,10 @@ class Holesky extends Testnet implements EthereumNetwork {
   // https://chainlist.org/chain/17000
   chainId = 17000;
   batcherContractAddress = '0x8ae286c75a339ffabaed3a22088c52eb4f589780';
-  forwarderFactoryAddress = '0x31f17917ea6132c9a2265f514970492bae3ed668';
-  forwarderImplementationAddress = '0x1a924814eaeee4aee5d687c8cc1ba57eb8db7477';
+  forwarderFactoryAddress = '0x23223fd7cfc5f34b20f38ebb9a489e1ff3c084d1';
+  forwarderImplementationAddress = '0xd4c24325b5ef514fbb1ecc76a9bb4dcb57a8341e';
+  walletFactoryAddress = '0xe9c16033e01bf9f94bf9f99fc0fefcfb8b124992';
+  walletImplementationAddress = '0xf8360c213e4f44cce03e48cb9281254de2a7a566';
```

Comment:
It seems like only forwarderFactoryAddress and forwarderImplementationAddress are required here in sdk

Reply:
I added those similar to polygon, maybe it is specific to polygon. Removed wallet and walletFactory contracts